packages/metal-state/src/Config.js

```diff
@@ -110,12 +110,82 @@ function mergeConfig(context, config) {
 	return obj;
 }
 
-// Add all validators to `Config`.
-const fnNames = Object.keys(validators);
-fnNames.forEach(
-	name => Config[name] = function() {
+/**
+ * Adds primitive type validators to the config object.
+ * @param {string} name The name of the validator.
+ */
+function setPrimitiveValidators(name) {
+	Config[name] = function() {
 		return this.validator(validators[name]);
-	}
-);
+	};
+}
+
+setPrimitiveValidators('any');
```

Comment:
Would it make more sense to make these (and `shapeOf`, `oneOfType`, etc.) just be methods on the `Config` as defined above? As oppose to declaring the object, and then mutating it down here over and over.

The implementation of each method could still just call some helper for the behavior that is shared.

Another idea would be to declare another object, something like `ConfigValidators`, and just merge the two objects at the end.